cube_81183_tag_data.toml

```diff
@@ -44,7 +44,7 @@ tags = ["Counters - Enabler","Keyword","Spirits - Enabler","ETB - Enabler","Tier
 mana_cost = ["1","W","R"]
 color_identity = "RW"
 types = ["creature"]
-tags = ["Signpost","Heroic - Payoff","Keyword","Tier 1"]
+tags = ["Signpost","Heroic - Payoff","Keyword","Tier 1","Tokens - Payoff"]
```

Comment:
Agreed, I've started to look at tagging for Galaxy Brain Cube and it definitely will require an expansion of the tagging system. So far my two ideas are:

1). Pretty much exactly what you described - a tag that's only active if you have enough of some other thing. There's a fair number of these cases that I've just fudged like this for now in this cube, but to generalize we'll need that. The interesting thing here that needs to be worked out imo is how to calculate the value of such an "X:Y" payoff given your current quantities of X and Y. But yea, this is needed I think.

2). Tagging which can use conditions expressed in some kind of query syntax, similar to scryfall, e.g. to allow the tag to specify red spells only for PyroGoggles (right now I had to make another tag for red spells to make PyroGoggles work correctly). This one is more for convenience but it might become necessary given how many cards have card property based conditions in the other cube.